app/pdfs/passengers_PDF.rb

```diff
@@ -0,0 +1,34 @@
+require 'prawn/table'
+
+class PassengersPDF < Prawn::Document
+  def initialize(passengers, filters)
+    super(page_layout: :landscape, page_size: "A3" )
+    content_width = bounds.width - 10
```

Comment:
I think that this doesn't do anything. It just sets a local variable while initializing, but that variable is discarded.